Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/Code.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class Code : QueryStringAuthorization
+    {
+        public override AuthorizationType AuthorizationType => AuthorizationType.Code;
+        public override string queryKeyName { get; protected set; } = "code";
```

Comment:
Waarom is deze public? Ik zou verwacht hebben dat dit geencapsuleerd is in deze class, als externe classes hiermee doen dan vraag ik mij af of dat niet een functie in deze value object moet zijn

Reply:
Na het lezen van de rest van de code zie ik dat dit nodig is voor de 3D Tiles handler, niet sjiek maar wel noodzakelijk voor nu